src/main/java/it/pagopa/interop/probing/scheduler/scheduler/ScheduledTasks.java

```diff
@@ -41,6 +47,8 @@ public void scheduleFixedDelayTask() {
       for (EserviceContent service : response.getContent()) {
         try {
           servicesSend.sendMessage(service);
+          eserviceClient.updateLastRequest(service.getEserviceRecordId(),
+              ChangeLastRequest.builder().lastRequest(OffsetDateTime.now()).build());
```

Comment:
correct me if I'm wrong, but I think the UTC timezone is not globally set in this service, so you need to pass the timezone to `now()` method

Reply:
Yes that is correct, thank you for pointing that out